swagger-spec/logs/definition.yaml

```diff
@@ -0,0 +1,68 @@
+# Log Model Definition
+type: object
+title: Log
+required:
+  - id
+properties:
+  id:
```

Comment:
Also, very minor but will be good for consistency, can you change the order of the keys for each property to be:
`type`
`format`
`readOnly`
`description`